pkg/email/template.go

```diff
@@ -0,0 +1,73 @@
+package email
+
+import (
+	"embed"
+	"html/template"
+	"net/url"
+	"path"
+	"strings"
+)
+
+const (
+	ResetPasswordPath = "/auth/resetpassword" //#nosec
+	InviteUserPath    = "/auth/users/create"  //#nosec
+
+	ResetPasswordEmailSubject = "Reset Password Request for your lakeFS account"
+	InviteUserWEmailSubject   = "You have been invited to lakeFS"
+)
+
+var (
+	//go:embed invite_user_template.html reset_email_template.html
+	f                  embed.FS
+	inviteTemplate, _  = f.ReadFile("invite_user_template.html")
+	resetTemplate, _   = f.ReadFile("reset_email_template.html")
```

Comment:
you are reading the data from embedded content - you can embed the data directly to a string

Reply:
Done